tests/test_custom_scalar.py

```diff
@@ -0,0 +1,68 @@
+import base64
+from typing import NewType
+
+import pytest
+
+import strawberry
+from strawberry.graphql import execute
+
+
+Base64Encoded = strawberry.scalar(
+    NewType("Base64Encoded", bytes),
+    serialize=base64.b64encode,
+    parse_value=base64.b64decode,
+)
+
+
+@strawberry.scalar(serialize=lambda x: 42, parse_value=lambda x: Always42())
+class Always42:
+    pass
+
+
+@pytest.mark.asyncio
```

Comment:
is there any particular reason why this test is async? I think we can use graphql sync here ;)

Reply:
The `execute` function is defined as `async`. I could wrap the call in an `asyncio.run`. Not sure if that is better?

Reply:
There's also graphql sync we could use, maybe we should make execute sync function 😊 
https://github.com/strawberry-graphql/strawberry/blob/master/tests/test_schema.py#L9

Reply:
I changed the test to use `graphql_sync`.